lib/rubocop/cop/style/inverse_methods.rb

```diff
@@ -28,6 +28,8 @@ module Style
       #   foo == bar
       #   !!('foo' =~ /^\w+$/)
       #   !(foo.class < Numeric) # Checking class hierarchy is allowed
+      #   # Blocks with guard clauses are ignored:
+      #   foo.select { |f| next if f.zero?; f != 1 }
```

Comment:
We do not recommend using semicolons within code, and we like to have our code examples follow as many best practices as possible. Please change this example to a multi-line block.

```ruby
#   foo.select do |f| 
#     next if f.zero?
#     f != 1
#   end
```

Reply:
done!